routes/stats.tsx

```diff
@@ -35,98 +35,81 @@ export const handler: Handlers<StatsPageData, State> = {
+      data: props.data.usersCounts,
+      borderColor: "#e85d04",
     },
     {
-      title: "Items created",
-      values: props.data.itemsCounts,
-      color: "#219ebc",
+      label: "Items created",
+      data: props.data.itemsCounts,
+      borderColor: "#219ebc",
     },
     {
-      title: "Votes",
-      values: props.data.votesCounts,
-      color: "#4338ca",
+      label: "Votes",
+      data: props.data.votesCounts,
+      borderColor: "#4338ca",
     },
   ];
 
-  const x = props.data.dates.map((date) =>
+  const max = Math.max(...datasets[0].data);
+
+  const labels = props.data.dates.map((date) =>
     new Date(date).toLocaleDateString("en-us", {
       month: "short",
       day: "numeric",
     })
   );
 
   return (
-    <main class="flex-1 p-4 grid gap-12 md:grid-cols-2">
-      {charts.map(({ color, title, values }) => {
-        const data = values.map((value) => Number(value));
-        const total = data.reduce(
-          (value, currentValue) => currentValue + value,
-          0,
-        );
-
-        return (
-          <div class="py-4">
-            <div class="text-center">
-              <h3>{title}</h3>
-              <p class="font-bold">{total}</p>
-            </div>
-            <Chart
-              container={{
-                class: "aspect-[2/1] mx-auto relative max-w-[100vw]",
-              }}
-              type="line"
-              options={{
-                plugins: {
-                  legend: { display: false },
-                },
-                interaction: {
-                  intersect: false,
-                },
-                scales: {
-                  x: {
-                    grid: { display: false },
-                  },
-                  y: {
-                    beginAtZero: true,
-                    grid: { display: false },
-                    max: Math.ceil(Math.max(...data) * 1.1),
-                    ticks: { stepSize: 1 },
-                  },
-                },
-              }}
-              data={{
-                labels: x,
-                datasets: [{
-                  label: title,
-                  data,
-                  borderColor: color,
-                  pointRadius: 0,
-                  cubicInterpolationMode: "monotone",
-                }],
-              }}
-            />
-          </div>
-        );
-      })}
+    <main class="flex-1 p-4 flex flex-col">
+      <h1 class="text-3xl font-bold">Stats</h1>
+      <div class="flex-1 relative">
+        <Chart
+          type="line"
+          options={{
+            maintainAspectRatio: false,
+            interaction: {
+              intersect: false,
+              mode: "index",
+            },
+            scales: {
+              x: {
+                max,
+                grid: { display: false },
+              },
+              y: {
+                beginAtZero: true,
+                grid: { display: false },
+                ticks: { stepSize: 1 },
```

Comment:
**suggestion:** as mentioned in https://github.com/denoland/saaskit/pull/343#discussion_r1252614079, we should be able to replace `{ stepSize: 1 }` with `{ precision: 0 }` to only have integers.

Reply:
Fixed.